### PR TITLE
enable remembering users

### PR DESCRIPTION
**Pull request by MaximilianMeister**
fixes bsc#1062184

Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

File: db/migrate/20171027115228_add_remember_token.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddRememberToken < ActiveRecord::Migration
+  def change
+    add_column :users, :remember_token, :string, :limit => 150, :index => true
```
Comment (ereslibre) [inline db/migrate/20171027115228_add_remember_token.rb]: I think we could place the column in a better place: `after: :reset_password_sent_at`, what do you think?

Comment (MaximilianMeister) [reply db/migrate/20171027115228_add_remember_token.rb]: no strong opinion on that, i updated it

Comment (ereslibre) [reply db/migrate/20171027115228_add_remember_token.rb]: The schema has not been updated :)

Comment (MaximilianMeister) [reply db/migrate/20171027115228_add_remember_token.rb]: the schema is not used in production anyways. and i ran the migration again and the result is the same in `schema.rb`

Comment (ereslibre) [reply db/migrate/20171027115228_add_remember_token.rb]: If you had already ran it yes, schema won't change. If you rerun the migration you'll see the column at the right position in the schema.rb file. 

We do use it the first time Velum runs (we do a load schema).

Comment (ereslibre) [reply db/migrate/20171027115228_add_remember_token.rb]: BTW just saw that the syntax of the hash is not unified, maybe that's the issue.

Comment (MaximilianMeister) [reply db/migrate/20171027115228_add_remember_token.rb]: i rolled back and then re-ran it of course. but still there was no change. the hash syntax i dont think is the issue, but i can try again

**Pull request merged**